apps/full-stack-tests/src/components/properties/ErrorHandling.test.tsx

```diff
@@ -32,6 +32,7 @@ describe("Learning snippets", () => {
     });
 
     it("handles errors", async function () {
+      const consoleStub = sinon.stub(console, "error").callsFake(()=> {});
```

Comment:
Maybe it is worth specifying which error we are trying to stub? (same with other console.error stubs)

Reply:
I don't see a point to it as the error message is generated most of the time, we don't know it's format (object, string, other). If we have any other errors within the test in the future the test would fail and if we need to debug it, you can just comment it out.

Reply:
Does logging to console also cause a warning in tests?

Reply:
Logging itself does not cause an external warning, but these tests expect an error to be thrown and handled and as a byproduct it also logs the error.

Reply:
I think this is resolved with https://github.com/iTwin/presentation/pull/799#discussion_r1886749785